analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -790,13 +789,106 @@ def add_arguments_to_parser(parser):
                                default=argparse.SUPPRESS,
                                help="Emit a warning instead of an error when "
                                     "an unknown checker name is given to "
-                                    "either --enable or --disable.")
+                                    "either --enable, --disable,"
+                                    "--analyzer-config and/or "
+                                    "--checker-config.")
 
     logger.add_verbose_arguments(parser)
     parser.set_defaults(
         func=main, func_process_config_file=cmd_config.process_config_file)
 
 
+def is_analyzer_config_valid(analyzer_conf: List[AnalyzerConfig]) -> bool:
+    """
+    Ensure that the analyzer_config parameter is set to a valid value
+    by verifying if it belongs to the set of allowed values.
+    """
+    if not isinstance(analyzer_conf, list):
+        return False
```

Comment:
Shouldn't we emit an error message here if we terminate CodeChecker?

Reply:
To be removed. Will be verified elsewhere.